styles.css

```diff
@@ -0,0 +1,695 @@
+  --font-md-3: 20px;
+  --font-lg: 24px;
+  --font-xl: 32px;
+  --font-1-5xl: 48px;
+  --font-2xl: 72px;
+  --font-family-primary: "Poppins", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+button,
+input,
+select,
+optgroup,
+textarea {
+  outline: 0;
+  border: 0;
+  font-family: var(--font-family-primary);
+  color: var(--text-color);
+}
+
+body {
+  color: var(--text-color);
+  font-size: var(--font-md);
+  font-family: var(--font-family-primary);
+  overflow-x: hidden;
+}
+
+/* CSS HEADER */
+.header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 50px 0;
+  z-index: 10;
+}
+
+.header-inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+}
+
+.logo a {
+  font-size: 19px;
+  display: flex;
+  align-items: center;
+  color: var(--white);
+}
+
+.logo a span {
+  margin-left: 22px;
+}
+
+.nav-menu {
+  display: flex;
+  align-items: center;
+  gap: 0 96px;
+}
+
+.nav-link {
+  color: var(--primary-color);
+  font-size: var(--font-md-2);
+  font-weight: 700;
+}
+
+.header-extra {
+  display: flex;
+  align-items: center;
+  gap: 0 24px;
+}
+
+.extra-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+#hambuger {
+  display: none;
+}
+
+/* BANNER */
+.section-banner.section {
+  background: url(./images/Fashion-home-image.png) no-repeat center/cover;
+  height: 800px;
+  position: relative;
+  width: 100%;
+  padding: 0;
+}
+
+.overlay {
+  background: linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0.8) 0%,
+    rgba(0, 0, 0, 0.5) 100%
+  );
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
+
+.container.banner-inner {
+  z-index: 3;
+  position: relative;
+  height: 100%;
+  width: 100%;
+  padding-left: 111px;
+}
+
+.banner-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+}
+
+.banner-title {
+  font-size: var(--font-2xl);
+  color: var(--white);
+  max-width: 403px;
+  line-height: 1;
+  margin-bottom: 24px;
+}
+
+.banner-title span {
+  color: var(--primary-color);
+}
+
+.content-link {
+  display: flex;
+  align-items: center;
+}
+
+.content-link-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  background-color: var(--primary-color);
+  margin-right: 21px;
+}
+
+.content-link-text {
+  text-transform: uppercase;
+  color: var(--white);
+  font-size: var(--font-lg);
+  font-weight: 700;
+  text-decoration: underline;
+}
+
+.banner-policy {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 76%;
+  z-index: 3;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: var(--white);
+  border-top-right-radius: 70px;
+  padding: 38px 88px 24px 0;
+  max-width: 1326px;
+}
```

Comment:
tổ chức lại thứ tự `css`. em tham khảo [link](https://medium.com/swlh/better-ways-to-organise-css-properties-9a066e7ded62) này rồi áp dụng hết cho các file `css`.